RSSReader-Maruyama/DataManager/UserDataManager.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoginChecker.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class UserDataManager: NSObject {
+    
+    private let UD_USER_DATA_KEY = "userdata"
+    lazy var userDefault = UserDefaults.standard
+
+    
+    func isLoggedIn() -> Bool {
```

Comment:
```suggestion
    func exists() -> Bool {
```
isLoggedInで行っている処理を見直してください。
ログイン状態の確認ではなく、userDetaulsへデータが保存されているかどうかの確認ですよね。
関数名も実装内容に則った名前にしましょう。

Reply:
かしこまりました。